example/programs/topic-subscriber/src/main.rs

```diff
@@ -0,0 +1,50 @@
+mod models;
+
+use futures::StreamExt;
+use momento::{topics::Subscription, CredentialProvider, MomentoError, TopicClient};
+use std::{env, error::Error};
+use tracing::{error, info};
+
+use crate::models::MomentoModel;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error>> {
+    tracing_subscriber::fmt()
+        .with_max_level(tracing::Level::INFO)
+        .with_target(false)
+        .json()
+        .init();
+
+    let momento_key = env::var("MOMENTO_API_KEY").expect("MOMENTO_API_KEY is required");
+    // create a new Momento client
+    let topic_client = TopicClient::builder()
+        .configuration(momento::topics::configurations::Laptop::latest())
+        .credential_provider(CredentialProvider::from_string(momento_key).unwrap())
+        .build()?;
```

Comment:
We have a `from_env_var` method on the credential provider for convenience:
```suggestion
    let topic_client = TopicClient::builder()
        .configuration(momento::topics::configurations::Laptop::latest())
        .credential_provider(CredentialProvider::from_env_var("MOMENTO_API_KEY")?)
        .build()?;
```

Reply:
Oh I didn't know about this.  I'll make the update and repush.

Reply:
@anitarua changes applied and pushed into my branch.